package.json

```
@@ -1,9 +1,12 @@
 {
   "name": "@bedrock/test",
-  "version": "8.0.6-0",
+  "version": "9.0.0-0",
   "type": "module",
   "description": "Bedrock test",
   "main": "./lib/index.js",
+  "engines": {
+    "node": ">=12"
```

This should be 14 since mocha 10 drops 12 support: https://github.com/mochajs/mocha/blob/master/CHANGELOG.md.
```suggestion
    "node": ">=14"
```